thinkcompany/think-ui-library
# TCR-90 header update

### Feature Status
- []  In Progress

### Description of Changes
- Revised markup and styles for redesigned header. 
- I couldn't figure out a decent workaround for the Storybook js issue we discussed, so I'll file a bug on it. To compensate, I've been clicking on a different component first, then go back to the Header so the navigation toggles as expected.

[[TCR-90](https://thinkcompany.atlassian.net/browse/TCR-90)]

#### Screenshots
<img width="1365" alt="Screen Shot 2021-02-18 at 5 00 37 PM" src="https://user-images.githubusercontent.com/1825366/108514132-7beebd00-7291-11eb-8f12-bd92843a90c0.png">
<img width="1212" alt="Screen Shot 2021-02-18 at 5 00 57 PM" src="https://user-images.githubusercontent.com/1825366/108514135-7c875380-7291-11eb-849c-6f6201b04c16.png">
<img width="386" alt="Screen Shot 2021-02-18 at 5 04 27 PM" src="https://user-images.githubusercontent.com/1825366/108514136-7c875380-7291-11eb-9e15-912224a3d91a.png">


### How to Review
Outline the steps a reviewer should take to view & test code locally.

1. `git fetch && git checkout feature/TCR-90-header-update`
2. `npm start`
3. Open any other component
4. Navigate to **Components** > **Global** > **Header**
Navigation should open/close as expected

### Dependencies
List all NPM dependencies added or changed and explain why they were added.
If no dependencies were added or changed, this heading can be removed


### Merge Checklist:

### GIF Description
Share a fun gif that describes this PR & how it makes you feel.

![Welcome](https://media.giphy.com/media/cssnLfdR664M03kpBA/giphy.gif)
